common/lib/api-client/models/population.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  fields: {
+    date: '',
+    operational_capacity: '',
+    usable_capacity: '',
+    unlock: '',
+    bedwatch: '',
+    overnights_in: '',
+    overnights_out: '',
+    out_of_area_courts: '',
+    discharges: '',
+    free_spaces: '',
+    updated_by: '',
+    created_at: '',
+    updated_at: '',
+  },
+  options: {
+    cache: false,
```

Comment:
The default is false so you can drop the whole options object for this model.

Reply:
Removed property from model